ui/apps/dashboard/src/app/(dashboard)/env/[environmentSlug]/functions/[slug]/logs/NewReplayButton.tsx

```diff
@@ -0,0 +1,33 @@
+'use client';
+
+import React, { useState } from 'react';
+import { Button } from '@inngest/components/Button';
+
+import NewReplayModal from '@/app/(dashboard)/env/[environmentSlug]/functions/[slug]/logs/NewReplayModal';
+import ReplayIcon from '@/icons/replay.svg';
```

Comment:
Since this is new, perhaps we could already add it to the icons in the components package. That way it can be reused, and you no longer need to set the icon color for the button